components/eamxx/src/share/atm_process/atmosphere_process.hpp

```diff
@@ -115,7 +115,7 @@ class AtmosphereProcess : public ekat::enable_shared_from_this<AtmosphereProcess
   const ekat::Comm& get_comm () const { return m_comm; }
 
   // Return the parameter list
-  const ekat::ParameterList& get_params () const { return m_params; }
+  ekat::ParameterList& get_params () { return m_params; }
```

Comment:
See comment where this is used. Was previously unused, but now I'm using and need to set a param (nonconst function).